plugin/src/main/java/net/aufdemrand/denizen/Denizen.java

```diff
@@ -227,6 +228,11 @@
 // |   m@<data_variety_material> - fetches the material specified by Denizen's 'data variety' dMaterials
 // |   m@random - fetches a random material
 //
+// + ----- dTrade -----+
+// | object notation: trade@    can reference unique objects: no      can be notable: no
+// | constructors: ( <>'s represent non-static information and are not literal)
+// |   trade@trade - fetches an empty trade with a result of i@air and zero maximum uses

```

Comment:
This is pretty unhelpful info. Reference the dInventory constructors data above.